src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const list = document.querySelectorAll('[data-salary]');
+const listArray = [...list];
+
+function sortList() {
+  function compare(a, b) {
+    if (+a.dataset.salary < +b.dataset.salary) {
+      return 1;
+    }
+
+    if (+a.dataset.salary > +b.dataset.salary) {
+      return -1;
+    }
+
+    return 0;
+  }
```

Comment:
create fuction which returns converted string into numbers without ',' and '$'
f.e. getConvertedSalary

```suggestion
    return getConvertedSalary(a.dataset.salary) - getConvertedSalary(b.dataset.salary);
  }
```